src/kusto/kusto.egg-info/PKG-INFO

```diff
@@ -1,34 +0,0 @@
-Metadata-Version: 1.1

```

Comment:
Hi @haroldrandom , I remove these egg-info, otherwise it will automatically install kusto in dev mode.